packages/base/room.gts

```diff
@@ -501,6 +504,8 @@ export class RoomField extends FieldDef {
           if (attachedCardIds.length < cardDocs.length) {
             throw new Error(`cannot handle cards in room without an ID`);
           }
+          (cardArgs as any).clientGeneratedId =
```

Comment:
instead of casting to `any` lets just initialize `clientGeneratedId` to `undefined` when we create `cardArgs`

Reply:
or better yet, let's just type `cardArgs`